### PR TITLE
Migrate dForce space to ENS.

### PR DESCRIPTION
**Pull request by Skyge**
#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [ ] Add a custom domain for your space
- [x] Add an alias to migrate your space

### CODE REVIEW

File: spaces/aliases.json [hunk 1]
```diff
@@ -47,5 +47,6 @@
   "pillar": "pillarwallet.eth", 
   "yaxis": "yaxis.eth",
   "spaceswap": "spaceswap.eth",
-  "descience.eth": "people-dao.eth"
+  "descience.eth": "people-dao.eth",
+  "df": "dforcenetwork.eth"
```
Comment (gokaiorg) [inline spaces/aliases.json:51]: You will need to set your actual space. Do you mean dforce space?
```suggestion
  "dforce": "dforcenetwork.eth"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/dforcenetwork.eth

File: spaces/aliases.json (hunk 1 above)
Comment (Skyge) [reply spaces/aliases.json]: So the key `dforce` is the original space without the ENS, right?

**Review by gokaiorg — APPROVED** [review]

File: spaces/aliases.json (hunk 1 above)
Comment (gokaiorg) [reply spaces/aliases.json]: Yes, exactly. Your changes are good.

**Pull request merged**